src/features/peers/redux/reducer.ts

```diff
@@ -0,0 +1,26 @@
+import { RESET } from "../../../store/constants";
```

Comment:
So in order to remove this deep nesting of imports take a look at `baseUrl` for your tsConfig file.

with `./src` you'll be able to change this 'to store/constants'.

I'd only do this for imports directly from src level as having nested structure can be helpful when navigating through isolated file structures.